criteria/findability.md

```diff
@@ -0,0 +1,51 @@
+# Findability
+
+## Requirements
+
+* The codebase MUST be findable using a search engine by description of the problem it solves in natural language
+* The codebase MUST be findable using a search engine by codebase name
+* Maintainers SHOULD submit the codebase to relevant software catalogs
+* The codebase SHOULD have a website which describes the problem the codebase solves using the preferred jargon of different roles or professions
```

Comment:
Should this perhaps end with something like "... that are users of the codebase"? This would narrow it down a bit and not suggest that all or a random set of roles and professions need to be catered for. Or is that implied enough through the context anyway?

Reply:
take a look now?